uint8 n
---
int8 rc
